pkg/config/apm.go

```diff
@@ -105,6 +105,7 @@ func setupAPM(config Config) {
 	config.BindEnv("apm_config.telemetry.additional_endpoints", "DD_APM_TELEMETRY_ADDITIONAL_ENDPOINTS")
 	config.BindEnv("apm_config.obfuscation.credit_cards.enabled", "DD_APM_OBFUSCATION_CREDIT_CARDS_ENABLED")
 	config.BindEnv("apm_config.obfuscation.credit_cards.luhn", "DD_APM_OBFUSCATION_CREDIT_CARDS_LUHN")
+	config.BindEnvAndSetDefault("apm_config.debug_server.port", 5012, "DD_APM_DEBUG_SERVER_PORT")
```

Comment:
Port 5012 bc the port range from 5000 to 5009 is used by the agent, the cluster agent and cluster check runner. The security agent's expvar port is 5011, so 5012 is just 5011+1.
5012 doesn't seem to be a registered / known port ([source](https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers)).